audio-worklet/src/bitcrusher-script-processor.js

```diff
@@ -76,7 +78,7 @@ class Bitcrusher {
    */
   processBuffer_(reduction, bitDepth, inputBuffer, outputBuffer) {
     if (reduction < 1) 
-      console.error("The minimum reduction rate is 1.");
+      console.error('The minimum reduction rate is 1.');
```

Comment:
What about checks for valid `bitDepth` values?  If you're checking for `reduction`, shouldn't you also check `bitDepth`?